local/load.php

```diff
@@ -11,6 +11,16 @@
 
 // Initialize local development tools below.
 
+/*
+ * Enables Jetpack development/debug mode.
+ *
+ * Setting JETPACK_DEV_DEBUG to true allows Jetpack features to run in a local development environment,
+ * bypassing certain production checks and enabling debugging tools. This should only be enabled for local development.
+ */
+if ( ! defined( 'JETPACK_DEV_DEBUG' ) ) {
+	define( 'JETPACK_DEV_DEBUG', true );
+}
```

Comment:
You don't want to enable dev mode on all sites ; you'll be putting all features that rely on a WordPress.com connection on pause on all connected sites. This should be something you only run in your development environment to allow you to test things out.

Reply:
The `local` folder is only available in local (dev) envs, it will be deleted before pushed to live systems.